portfolio/src/main/webapp/scripts/script.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Fetches data from the server and adds it to the DOM.
+ */
+ function fetchData() {
+  fetch('/data').then(response => response.json()).then((msgs) => {
+    const msgsTest = document.getElementById('fetched-data');
+    msgsTest.innerText = msgs;
```

Comment:
this makes all the msgs squished into one, but I suppose it will be fixed in the next PR

Reply:
yup, I intend to make a Message class as soon as I can